CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ project(wav2letter++)
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_LINKER_WRAPPER_FLAG "-Xlinker" " ")
```

Comment:
@tbfly — it's not clear to me why these are needed: can you explain? Which version of CMake/gcc/ld,gold are you using that this is required?